ui/modal/create_watch_only_modal.go

```diff
@@ -110,6 +110,11 @@ func (cm *CreateWatchOnlyModal) Handle() {
 			cm.Dismiss()
 		}
 	}
+
+	if cm.modal.BackdropClicked() {
+		cm.Dismiss()
+		cm.RefreshWindow()
```

Comment:
This window refresh should be done here instead https://github.com/planetdecred/godcr/blob/master/ui/window.go#L207